getj2me

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+node --harmony ./_getj2me
```

Comment:
Can't we just rename ```_getj2me``` to ```getj2me``` and execute that file directly? What's the reasoning for having this as a runner for it? Can you add a comment in this file as to why we need it?